app/src/main/AndroidManifest.xml

```diff
@@ -348,6 +348,7 @@
                 <data android:host="frw-link.lilico.app"/>
                 <data android:host="fcw-link.lilico.app"/>
                 <data android:host="link.wallet.flow.com"/>
+                <data android:scheme="tg"/>
```

Comment:
I don't think this is right, I think we put `tg` deeplink schema in here, it will register our app as tg://